modules/UI/util/MessageHandler.js

```diff
@@ -421,9 +421,12 @@ const messageHandler = {
      *
      * @param {object} props - The properties to pass to the
      * showErrorNotification action.
+     * @param {boolean} showSupportLink - Whether or not the notification should
+     * display a link to open the support page. Defaults to true.
+     * @returns {void}
      */
-    showError(props) {
-        APP.store.dispatch(showErrorNotification(props));
+    showError(props, showSupportLink = true) {
```

Comment:
What was the idea behind moving the hideErrorSupportLink out of the props as a separate param? If we do that for the supportLink why keep the other 2 in the props?

Reply:
There's a link that needs to display for the firefox screenshare extension. It sounded like in the ticket you wanted to be able to display the link somehow to download the firefox screenshare extension. Because the support link set the precedent for opening an external link, I added more generic props for adding a button with a link. With that approach, it didn't make sense to have a separate button specifically for the support link so I moved that logic to use the more generic props.